src/quo2/components/drawers/documentation_drawers/view.cljs

```diff
@@ -0,0 +1,33 @@
+(ns quo2.components.drawers.documentation-drawers.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.components.drawers.documentation-drawers.style :as style]
+            [react-native.core :as rn]
+            [quo2.components.buttons.button :as button]))
+
+(defn view
+  "Options
+   - `title` Title text
+   - `cta?` Show CTA button
```

Comment:
we don't need `cta?` prop, you can just do `when on-press-cta ...` below

Reply:
`cta?` is used to show/hide the button.

Reply:
yes but we only need the button when there is an on-press action 🤔 
Maybe I have it wrong :)

Reply:
Yes, we can do that. In the component design it's a 'Read more' button and when clicked it will show the full content and hide the button. So I thought it's better to add a separate prop to control the visibility of the button. But we can achieve it using `on-press`, should I remove `cta?`?

Reply:
Ah yea, you're right. Let's leave it as is so 👌